src/include/stir/recon_buildblock/BackProjectorByBin.h

```diff
@@ -111,14 +112,42 @@ class BackProjectorByBin :
 		   const RelatedViewgrams<float>&,
 		   const int min_axial_pos_num, const int max_axial_pos_num,
 		   const int min_tangential_pos_num, const int max_tangential_pos_num);
+#endif
+ //! project whole proj_data into the volume
+ /*! it overwrites the data already present in the volume */
+ virtual void back_project(const ProjData&, int subset_num = 0, int num_subsets = 1);
+
+ /*! \brief projects the viewgrams into the volume
+  it adds to the data already present in the volume.*/
+void back_project(const RelatedViewgrams<float>&);
+
+ /*! \brief projects the specified range of the viewgrams into the volume
+  it adds to the data already present in the volume.*/
+void back_project(const RelatedViewgrams<float>&,
+          const int min_axial_pos_num, const int max_axial_pos_num);
+
+ /*! \brief projects the specified range of the viewgrams into the volume
+   it adds to the data already present in the volume.*/
+void back_project(const RelatedViewgrams<float>&,
+          const int min_axial_pos_num, const int max_axial_pos_num,
+          const int min_tangential_pos_num, const int max_tangential_pos_num);
 
+ /*! \brief tell the back projector to start accumulating into a new image*/
+ virtual void start_accumulating_in_new_target();
+
+ /// Get output
```

Comment:
```suggestion
 /*! \brief Get output
 
   This will overwrite the array-content of the argument with the result of all backprojections since calling `start_accumulating_in_new_target()`. Note that the argument has to have the same characteristics as what was used when calling `set_up`.
 */
```